### ISSUE TITLE
Could you mention accuracy rates for each model in documentation

### ISSUE THREAD

**Issue by ajithvcoder** [sgryco/keras_tinyimagenet#1]
accuracy rates for each model with tiny image dataset

**Comment by sgryco** [comment]
The best result was achieved with using pre-trained networks: Inception v3 with 71.4% accuracy and Vgg16 gave 61.7%. The network MirrotNet-4 introduced in this project reached 48.9% accuracy trained from scratch. More details are present in the report.

**Comment by ajithvcoder** [comment]
thanks for the info
